saltproc/input_schema.json

```diff
@@ -54,7 +54,13 @@
                             "exec_path": {
                                 "default": "sss2"},
                             "template_input_file_path": {
-                                "pattern": "^(.\\/)*(.*)$"}
+                                "pattern": "^(.\\/)*(.*)$"},
+                            "zaid_convention": {
+                                "description": "ZAID naming convention for nuclide codes. 'serpent': The third digit in ZA for nuclides in isomeric states is 3 (e.g. 47310 for for Ag-110m). 'mcnp': ZA = Z*1000 + A + (300 + 100*m). where m is the mth isomeric state (e.g. 47510 for Ag-110m). 'nndc': Identical to 'mcnp', except Am242m1 is 95242 and Am242 is 95642",
+                                "type": "string",
+                                "enum": ["serpent", "mcnp", "nndc"],
+                                "default": "mcnp"
```

Comment:
Could a default `zaid_convention` be set based on the `codename` provided (i.e. if `codename` serpent is provided, then the `zaid_convention` would default to `serpent` if none were provided)?

Reply:
You could, but it's very much possible to use the `nndc`, `serpent`, and `mcnp` conventions in Serpent depending on where you're getting your cross section libraries from.